src/gui/CMakeLists.txt

```diff
@@ -258,10 +256,6 @@ add_library(updater STATIC ${updater_SRCS})
 target_link_libraries(updater ${synclib_NAME} ${updater_DEPS} Qt5::Widgets Qt5::Network Qt5::Xml)
 target_include_directories(updater PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-set_target_properties( ${APPLICATION_EXECUTABLE} PROPERTIES
-        RUNTIME_OUTPUT_DIRECTORY  ${BIN_OUTPUT_DIRECTORY}
-)
```

Comment:
See comment above regarding `RUNTIME_OUTPUT_DIRECTORY`.

Reply:
ecm sets it globaly for some platforms, for other platforms it should not be needed but I'll check